recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "coiled-runtime" %}
 {% set version = "0.2.0" + environ.get("VERSION_SUFFIX", '') %}
-{% set dask_version = environ.get("DASK_VERSION", "2022.10.2") %}
-{% set distributed_version = environ.get("DISTRIBUTED_VERSION", "2022.10.2") %}
+{% set dask_version = environ.get("DASK_VERSION", "2022.6.0") %}
```

Comment:
@jrbourbeau since we are doing a patch, should we bump to 2022.6.1 see https://github.com/coiled/coiled-runtime/issues/306
cc: @hendrikmakait

Reply:
Let's stick with 2022.6.0 to keep this PR as minimal as possible. I expect we'll push out another release soon where we update other package versions